pkg/runtime/interfaces/application_configuration.go

```diff
@@ -93,6 +191,48 @@ type WorkflowExecutorConfig struct {
 	AccountID string `json:"accountId"`
 }
 
+func (a *AWSWorkflowExecutorConfig) GetRegion() string {
+	return a.Region
+}
+
+func (a *AWSWorkflowExecutorConfig) GetScheduleScheduleQueueName() string {
+	return a.ScheduleQueueName
+}
+
+func (a *AWSWorkflowExecutorConfig) GetAccountID() string {
+	return a.AccountID
+}
+
+// FlyteWorkflowExecutorConfig specifies the workflow executor configuration for the native flyte scheduler
+type FlyteWorkflowExecutorConfig struct {
```

Comment:
please add default values for all these values.

cc @katrogan / @EngHabu any new config should have a default value, else we have to go through a justification

Reply:
did you add the default values?

Reply:
@pmahindrakar-oss default values? Where did you add them. you can see an example [here](https://github.com/flyteorg/flyteadmin/blob/master/auth/config/config.go#L52)

Reply:
Default values have to be done on this change, as it will break others

Reply:
Done